.travis.yml

```diff
@@ -1,42 +0,0 @@
-# This is a special configuration file to run tests on Travis-CI via
-# GitHub notifications when changes are committed.
-#
-# For technical details, see http://travis-ci.org/
-
-language: python
-cache: pip
-jobs:
-  include:
-    - stage: test
-      python: "3.6"
-    - stage: test
-      python: "3.7"
-    - stage: test
-      python: "3.8"
-
-before_install:
- - "if grep '\t' *.py *.rst; then echo 'Tabs are bad, please use four spaces.'; false; fi"
- - "if grep -n -r '[[:blank:]]$' *.py *.rst; then echo 'Please remove trailing whitespace.'; false; fi"
- - pip install --upgrade pip setuptools
- - pip install flake8 flake8-docstrings restructuredtext-lint black
- - echo "Using restructuredtext-lint to check documentation"
- - restructuredtext-lint *.rst
- - echo "Using flake8 to check Python code"
- - flake8 *.py
- - echo "Using black to check Python code"
- - black --check *.py
- - echo "Using flake8 to check tests (with lots of exclusions)"
- - flake8 tests/
-
-install:
- - python setup.py install
-
-script:
- - echo "On this machine \$LANG=$LANG"
- - echo "Checking our own code passes"
- - flake8 --select BLK *.py
```

Comment:
Note, I was not able to elegantly port this one line to be tested via Tox. This doesn't actually matter, since all of the source code is already verfied to be Black-compliant via explicit invocation of `black --check`.

Since the glob expansion of `*` is done by the shell (and is not done by Tox), we'd need to pass an explicit `--exclude=tests` option to `flake8`. However, this would overwrite the default excludes of `.git` and lots of other potential directories, and I thought it was too awkward to start explicitly re-listing all of those default excludes.

If you really want to verify that running the current version of flake8-black on its own source code via `flake8 --select BLK`, then it's probably best to put all the source code into an `flake_black` subdirectory (which is generally a Python best practice anyway), to make it easier to only invoke `flake8` on that directory.